package.json

```diff
@@ -13,7 +13,8 @@
     "lib"
   ],
   "scripts": {
-    "test": "jest --coverage && standard",
+    "test": "npx jest --coverage && standard",
+    "lint": "npx eslint index.ts bin/smee.js test/index.test.ts",
```

Comment:
I would recommend a wildcard expression, so that, if/when new modules are added they get picked up automagically and no one has to remember to add them here....